lib/libc/armstdc/include/limits.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Intel
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_LIB_LIBC_ARMSTDC_INCLUDE_LIMITS_H_
+#define ZEPHYR_LIB_LIBC_ARMSTDC_INCLUDE_LIMITS_H_
+
+#include_next <limits.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define PATH_MAX 256
```

Comment:
Where did this particular value come from? picolibc uses 1024, fwiw.

Reply:
I grabbed the value from lib/libc/minimal/include/limits.h

Reply:
Updated commit message to explain where the value came from.

Reply:
ok, seems fine to me, although a bit small by comparison with other systems.